### ISSUE TITLE
Did not received XLM because no memo was indicated

### ISSUE THREAD

**Issue by rizbee** [orbitlens/stellar-expert-explorer#294]
Need your help. I did not received my XLM from exodus. Please see details below:

Transaction ID: 5fd8ebfc2d40ad9adc416217f76cd599a890565356de2c6abe02f5b7fe727ec9

Address: GBLJ5ZBZKZXQMXUEABO4FWLR7YWXYOJFLDISAE3TSHT4XVOWLJYBBXCS

Memo: 19732606

Amount sent: 1284.4969 XLM

**Comment by cjcase05** [comment]
I have the same issue... Cant figure out a solution..

**Comment by jcorrierefit** [comment]
I have the same issue for two transactions. I XLM twice from my ledger to my coinbase account and forgot the memo. Didn't receive my xlm onto my coinbase. Can you help?

**Comment by Gmon-crypto** [comment]
> I have the same issue for two transactions. I XLM twice from my ledger to my coinbase account and forgot the memo. Didn't receive my xlm onto my coinbase. Can you help?

Same here!I have opened two cases with coinbase,haven’t heard anything back from them.If a memo id is required,why does it let you proceed without it???That makes it optional to me...

**Comment by Gmon-crypto** [comment]
I don’t think this is actual support for XLM crypto from this site.Try here for some help https://medium.com/stellar-community/help-i-forgot-my-stellar-memo-d62b3cc9c2f7

**Comment by evamai2013** [comment]
I have same problem has anyone sorted out there problem I forgot memo

**Comment by evamai2013** [comment]
2500 xlm 🤦🏻‍♂️
